src/mem/vmem/vmem_nommu.h

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file
+ * @brief Virtual memory subsystem
+ *
+ * @date 05.10.2012
+ * @author Anton Bulychev
+ */
+
+#ifndef VMEM_NOMMU_H
+#define VMEM_NOMMU_H
+
+#include <hal/mmu.h>
+#include <sys/mman.h>
+
+#define VMEM_PAGE_WRITABLE    (1 << 0)
+#define VMEM_PAGE_CACHEABLE   (1 << 1)
+#define VMEM_PAGE_USERMODE    (1 << 2)
+/* Used on sparc LEON3 */
+#define VMEM_PAGE_EXECUTABLE  (1 << 3)
+
+typedef uint32_t vmem_page_flags_t;
+
+static inline vmem_page_flags_t prot_to_vmem_flags(uint32_t flags) {
```

Comment:
This function is duplicated in vmem.h

Reply:
Yes, but I suggest to fix it in the  future